test/mocha-setup.js

```diff
@@ -1,4 +1,3 @@
 if (typeof process === 'undefined')
   global.process = { env: {} };
 process.env.SYSTEM_TRACING = 1;
```

Comment:
Not related to this PR, but worth noting this line can also be removed now since `SYSTEM_PRODUCTION` as "not tracing" is the current rule.